apps/system/js/activities.js

```diff
@@ -1,7 +1,23 @@
 'use strict';
-/* global ActionMenu, applications, ManifestHelper, DefaultActivityHelper */
+/* global ActionMenuSystemDialog, BaseModule, applications,
+  ManifestHelper, DefaultActivityHelper */
 
-(function(exports) {
+(function() {
+
+
+  var Activities = function Activities() {
+    this.actionMenu = null;
+  };
+
+  Activities.EVENTS = [
+    'mozChromeEvent',
+    'appopened',
+    'applicationinstall'
+  ];
+
+  Activities.SUB_MODULES = [
+    'ActionMenuSystemDialog'
```

Comment:
Doing this here will load and start the dialog right away after activities are started. Is this what we expect? If not, let's remove it and lazy load the script on demand (this file, L206)